src/hubinfo.ts

```diff
@@ -0,0 +1,85 @@
+import * as Consts from "./consts";
+import { Hub } from "./hub";
+import { LPF2Hub } from "./lpf2hub";
+import { PoweredUP } from "./poweredup-node";
+import { toBin, toHex } from "./utils";
+
+function sanitizeString(s: string) {
+    return s.replace(/\0/g, " ").replace(/ +$/, "");
+}
+
+export function main(argv: string[]) {
+    let portInfo = false;
+    let quiet = false;
+    for (const arg of argv) {
+        switch (arg) {
+            case "-p":
+            case "--portinfo":
+                portInfo = true;
+                break;
+            case "-q":
+            case "--quiet":
+                quiet = true;
+                break;
+            default:
+                console.log("Usage: lpf2-hubinfo [-p|--portinfo] [-q|--quiet]");
```

Comment:
The process never quits, likely as Noble is still running in the background. Can you add something like `process.exit(0);`?